include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -388,5 +382,18 @@ NOTE(nonescapable_field_of_escapable, none,
      "escapable record %0 cannot have non-escapable field '%1'",
      (const clang::NamedDecl *, StringRef))
 
+ERROR(invalid_conditional_attr, none,
+      "attribute '%0' is invalid because it is only supported in "
```

Comment:
nit: I think the other annotations don't add quotation marks around the annotation/attribute.

Also, to be really nit-picky, what the user writes is an _annotation_ (implemented via a macro), which expands to an attribute; we should word the diagnostic to be consistent with that vocabulary:

```suggestion
      "annotation %0 is invalid because it is only supported in "
```